### PR TITLE
Global drafts documentation

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for:
- https://github.com/mattermost/mattermost-server/pull/20614
- https://github.com/mattermost/mattermost-webapp/pull/8869

### CODE REVIEW

**Comment by esethna** [comment]
@Rishselva can you help take a first pass on reviewing this documentation?

File: source/channels/send-messages.rst [hunk 1]
```diff
@@ -29,6 +29,21 @@ Compose a message by typing into the text box at the bottom of Mattermost. Press
 
       In Mattermost versions up to v5.39, you can configure Mattermost to press :kbd:`Enter` on Windows or Linux, or :kbd:`↵` on Mac to insert new lines, and press :kbd:`Ctrl` :kbd:`Enter` on Windows or Linux, or :kbd:`⌘` :kbd:`↵` on Mac to send messages instead. Select the three horizontal lines at the top of the channel sidebar (also known as a hamburger menu) to go to **Account Settings**, then select **Advanced > Send messages on CTRL+ENTER**.
   
+Draft messages
+--------------
+
+From Mattermost v7.6, when composing new messages, it's easy to return to a message in progress later. Draft messages are added to a **Drafts** view available at the top of the channel sidebar unless your system admin `disables global draft functionality </configure/experimental-configuration-settings.html#allow-synchronized-drafts>`__.
```
Comment (esethna) [inline source/channels/send-messages.rst:35]: 7.7

Comment (esethna) [inline source/channels/send-messages.rst:35]: Actually the drafts tab will show no matter what the config setting is set to. The setting turns off whether the drafts are synced to the server/available across devices once mobile support is built

File: source/configure/experimental-configuration-settings.rst [hunk 2]
```diff
@@ -383,9 +383,22 @@ This setting enables the Apps Bar and moves all Mattermost integration icons fro
...
-| This feature's ``config.json`` setting is ``"EnableAppBar": false`` with options ``true`` and ``false``. |
-+----------------------------------------------------------------------------------------------------------+
++-------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalSettings.EnableAppBar": false`` with options ``true`` and ``false``. |
++-------------------------------------------------------------------------------------------------------------------------------+
+
+Allow synchronized drafts
+-------------------------
+
+This setting saves in-progress message drafts in the Drafts view via the channel sidebar, and synchronizes draft messages across all supported Mattermost clients.
```
Comment (esethna) [inline source/configure/experimental-configuration-settings.rst:393]: This setting ~~saves in-progress message drafts in the Drafts view via the channel sidebar, and~~ synchronizes draft messages across all supported Mattermost clients.

**Review by esethna — CHANGES_REQUESTED** [review]

File: source/channels/send-messages.rst (hunk 1 above)
Comment (justinegeffen) [inline source/channels/send-messages.rst:35]:
```suggestion
From Mattermost v7.7, when composing new messages, it's easy to return to a message in progress later. Draft messages are added to a **Drafts** view available at the top of the channel sidebar unless your system admin `disables global draft functionality </configure/experimental-configuration-settings.html#allow-synchronized-drafts>`__.
```

File: source/configure/experimental-configuration-settings.rst (hunk 2 above)
Comment (justinegeffen) [inline source/configure/experimental-configuration-settings.rst:393]:
```suggestion
This setting synchronizes draft messages across all supported Mattermost clients.
```

File: source/channels/send-messages.rst [hunk 3]
```diff
@@ -29,6 +29,21 @@ Compose a message by typing into the text box at the bottom of Mattermost. Press
...
+Draft messages
+--------------
+
+From Mattermost v7.7, when composing new messages, it's easy to return to a message in progress later. Draft messages are added to a **Drafts** view available at the top of the channel sidebar unless your system admin `disables global draft functionality </configure/experimental-configuration-settings.html#allow-synchronized-drafts>`__.
+
+.. image:: ../images/Global-Drafts-Animated-GIF.gif
+  :width: 700
+  :alt: Global drafts makes it easy for you to find all messages in progress.
+
+By default, message drafts are synchronized on the Mattermost server AND accessible everywhere you access Mattermost, including a web browser or the desktop app with mobile support coming in a future release.
```
Comment (justinegeffen) [inline source/channels/send-messages.rst:41]:
```suggestion
By default, message drafts are synchronized on the Mattermost server and are accessible everywhere you access Mattermost, including a web browser or the desktop app with mobile support coming in a future release. 
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
> @Rishselva can you help take a first pass on reviewing this documentation?

I inadvertently removed @rishelva from the PR, and can't find them again. Sorry!

File: source/configure/experimental-configuration-settings.rst [hunk 4]
```diff
@@ -383,9 +383,22 @@ This setting enables the Apps Bar and moves all Mattermost integration icons fro
...
++-------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"ExperimentalSettings.EnableAppBar": false`` with options ``true`` and ``false``. |
++-------------------------------------------------------------------------------------------------------------------------------+
+
+Allow synchronized drafts
+-------------------------
+
+This setting synchronizes draft messages across all supported Mattermost clients.
+
+**True**: **(Default)** Global drafts are enabled, synchronized across clients, and draft messages are accessible through the Drafts view.
```
Comment (esethna) [inline source/configure/experimental-configuration-settings.rst:395]: True: Message drafts are saved on the server and may be accessed from different clients. Users may still disable server synchronization of draft messages in Account Settings.

File: source/configure/experimental-configuration-settings.rst [hunk 5]
```diff
@@ -383,9 +383,22 @@ This setting enables the Apps Bar and moves all Mattermost integration icons fro
...
++-------------------------------------------------------------------------------------------------------------------------------+
+
+Allow synchronized drafts
+-------------------------
+
+This setting synchronizes draft messages across all supported Mattermost clients.
+
+**True**: **(Default)** Global drafts are enabled, synchronized across clients, and draft messages are accessible through the Drafts view. 
+
+**False**: Global drafts are disabled.
```
Comment (esethna) [inline source/configure/experimental-configuration-settings.rst:397]: False: Draft messages are only stored locally on each users device

**Review by esethna — APPROVED** [review]
@cwarnermm looks good, I just modified the proposed wording for the config setting as the drafts item is still available even if that is set to false (it just doesn't sync on the server)

File: source/configure/experimental-configuration-settings.rst (hunk 5 above)
Comment (cwarnermm) [inline source/configure/experimental-configuration-settings.rst:397]:
```suggestion
**False**: Draft messages are only stored locally on each user's device.
```

File: source/configure/experimental-configuration-settings.rst (hunk 4 above)
Comment (cwarnermm) [inline source/configure/experimental-configuration-settings.rst:395]:
```suggestion
**True**: **(Default)** Message drafts are saved on the server and may be accessed from different clients. Users may still disable server synchronization of draft messages by going to **Settings > Advanced Settings**. 
```

**Pull request merged**